src/actions/application.js

```diff
@@ -48,6 +48,37 @@ function* loadMenu() {
   }
 }
 
+/**
+ * Gets all available content types.
+ */
+export const CONTENT_TYPES_REQUESTED = 'CONTENT_TYPES_REQUESTED';
```

Comment:
It feels really weird to put these reducers into application.js. Is there a reason this is not moved to a admin/content specific page?

Reply:
@dawehner exactly :+1: